### ISSUE TITLE
Is Jtwig dead? 

### ISSUE THREAD

**Issue by eXsio** [jtwig/jtwig#377]
Hello

I love Jtwig and it is my go-to templating engine every time I need one. I've noticed that the development is somewhat stagnated and no new features are added. Does it mean that Jtwig is dead? I sure hope not..

**Comment by ludovicmartin** [comment]
Hi
Same feelings...
Pull requests and bug reports never gets reply since months (years?).

**Comment by SBI-** [comment]
Seems to be the case, as the [official website](http://www.jtwig.org/) says that it's no longer maintained.

**Comment by eXsio** [comment]
Yup, it's officially dead. Now that docs and tutotials are offline the code is unusable for the newcommers and the deps won't be updated. Time to opt out.

I've already escaped to Pebble. It's very similar, simpler to extend, easy to contribute to and is still maintaned. And also (at least for my usecase) up to 3x faster (measured using JMH). 

Even if it goes offline like JTwig, it will be much easier to fork and maintain it.

**Comment by joao-de-melo** [comment]
Hi everyone, due to personal circumstances I can no longer ensure a good maintenance of Jtwig. It was a hard decision to make that I've been postponing for a while, but that is the painful and honest truth, I can no longer keep this project up.

However if anyone is interested in taking over or carry on with the project I can easily provide the resources and servers.

Thanks for everything though. In terms of alternatives, I do recommend pebble as a good alternative. 

Jtwig goal of mapping twig dynamics much dependent on PHP scripting nature made it able to be expended by the users of it but at the expense of a more complex backbone. Pebble has a more limited approach to extension, but I do think with a lot of benefits to namely simplicity (the eternal fight between template simplicity vs presentation logic encapsulation).

That is all. Feel free to reach out on my email if you have energy or ideas for the future of Jtwig: joao.silva.de.melo at gmail dot com

**Comment by ben-manes** [comment]
Can you please keep the docs up? You will still have users who are maintaining their usage, pending a migration. I appreciate the effort and understand disbanding the project, but removing documentation is a bit hostile.

**Comment by ludovicmartin** [comment]
+1
